### PR TITLE
Fix add variant to cart test after removing cart view

### PR DESCRIPTION
**Pull request by szczecha**
Fix test and remove cypress from dependencies in app/storefront

### CODE REVIEW

**Review by mmiszy — APPROVED** [review]

**Comment by mmiszy** [comment]
You need to run `pnpm i` to update the lockfile @szczecha

**Review by mmiszy — APPROVED** [review]

File: cypress/support/pages/product-page.ts [hunk 1]
```diff
@@ -12,47 +12,21 @@ export function openProductPage(product) {
...
-            .invoke("text")
-            .then((selectedVariantName) => {
-              cy.get(PRODUCT_ELEMENTS.variant)
-                .first()
-                .click()
-                .get(PRODUCT_ELEMENTS.addToCartButton)
-                .click()
-                .url()
-                .should("include", "/cart")
-                .get(CART.cartVariant)
```
Comment (karola312) [inline cypress/support/pages/product-page.ts:34]: Remove CART import it is no longer used

File: cypress/e2e/addVariantToCart.cy.ts [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-    cy.visit("/");
+    cy.visit("/").clearLocalStorage();
   });
 
   it("should select a variant and add to the cart SRS_0202", () => {
     const product = productsToSearch.productWithVariants;
 
     openProductPage(product);
-    addItemToCart(PRODUCT_ELEMENTS.productName);
+    addItemToCart();
```
Comment (karola312) [inline cypress/e2e/addVariantToCart.cy.ts:13]: It would be nice if at the end we check the cart icon - there should be a number of items in the cart

Comment (szczecha) [reply cypress/e2e/addVariantToCart.cy.ts:13]: In the future, there will be a mini cart and then the test will be checking if that product is in the cart.

**Pull request merged**